api/proto/config/storage.proto

```diff
@@ -106,3 +106,48 @@ message Drive {
   // A value of 0 will indicate that no resizing is required
   int64 maxsizebytes = 10;
 }
+
+
+message ContentTree {
+  UUIDandVersion uuidandversion = 1;
+  string dsId  = 2; // effectively pointer/key into dsConfigs
+
+  string URL = 3;
```

Comment:
Any conventions in how the URL and DSID are to be used together? Or just follow whatever we use currently for images?

Reply:
These are the same (with the same semantics) as in the existing Drive definition.